internal/connect/command_offset_status_describe.go

```diff
@@ -0,0 +1,176 @@
+package connect
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tidwall/pretty"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type serializedOffsetStatusDescribeOut struct {
+	Id               string           `json:"id" yaml:"id"`
+	RequestType      string           `json:"request_type" yaml:"request_type"`
+	RequestedAt      string           `json:"requested_at" yaml:"requested_at"`
+	Phase            string           `json:"phase" yaml:"phase"`
+	Message          string           `json:"message,omitempty" yaml:"message,omitempty"`
+	AppliedAt        string           `json:"applied_at,omitempty" yaml:"applied_at,omitempty"`
+	RequestedOffsets []map[string]any `json:"requested_offsets,omitempty" yaml:"requested_offsets,omitempty"`
```

Comment:
Will this ever be empty? Wondering why we're using `omitempty` here.

Reply:
yes, it will be empty in case of delete